TUM Campus App/ViewControllerProvider.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ViewControllerProvider.swift
+//  TUM Campus App
+//
+//  Created by Robert on 06.10.17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class ViewControllerProvider {
+    static private(set) var loginNavigationViewController: UINavigationController? = {
+        let storyboard = UIStoryboard(name: "Setup", bundle: nil)
```

Comment:
I'm still not a big fan of adding more Singletons. But what the hell, who cares?!?

Reply:
To be honest, me neither. However, I guess in this case it is fine, since we just kind of "bundle" the work that other parts of the code might have done in one place that is easy to get to. I guess in this case there is no really strong argument for or against either way.